docs/guides/networking/linode-network/ip-sharing-atlanta/index.md

```diff
@@ -0,0 +1,265 @@
+neighbor HOST remote-as external
+neighbor HOST capability extended-nexthop
+neighbor [DEFAULT_GW_IPV4] peer-group HOST
+address-family ipv4 unicast
+  network [ELASTIC_IP]/32 route-map [ROLE]
+  redistribute static
+exit-address-family
+route-map primary permit 10
+set large-community 63949:1:1
+route-map secondary permit 10
+set large-community 63949:1:2
+{{</ file >}}
+
+1.  Run the VTY shell:
+
+        sudo vtysh
+
+1.  Enter configuration mode:
+
+        conf t
+
+1.  Copy the contents of your template configuration file and paste them into the VTY shell.
+
+1.  Tell the VTY shell that you are done entering your configurations:
+
+        end
+
+1.  Write your configurations to VTY:
+
+        write
+
+1.  Verify that the configurations you entered were correctly written by showing VTY's running configuration:
+
+        show running-config
+
+1.  Exit out of the VTY shell:
+
+        q
+
+1.  Configure the Linode's interface(s) with the Elastic IP:
+
+    **Debian 10 & Ubuntu 18.04**
+
+    Edit your Linode's `/etc/network/interfaces` file with the following entries. Replace `[ELASTIC_IP]` with the Elastic IPv4 address:
+
+    {{< file >}}
+up   ip addr add [ELASTIC_IP]/32 dev eth0 label eth0
+down ip addr del [ELASTIC_IP]/32 dev eth0 label eth0
+{{</ file >}}
+
+    If you configured more than one Elastic IP on your Linode, you can add additional interface entries to your network interfaces configuration file as follows:
+
+    {{< file >}}
+up   ip addr add [ELASTIC_IP]/32 dev eth0 label eth0
+down ip addr del [ELASTIC_IP]/32 dev eth0 label eth0
+up   ip addr add [ELASTIC_IP]_2/32 dev eth0 label eth0
+down ip addr del [ELASTIC_IP]_2/32 dev eth0 label eth0
+{{</ file >}}
+
+    **Ubuntu 20.04**
+
+    Edit your Linode's `/etc/systemd/network/05-eth0.network` file by adding an `Address` entry for the Elastic IP. Replace `[ELASTIC_IP]` with the Elastic IPv4 address:
+
+    {{< file >}}
+[Match]
+Name=eth0
+...
+Address=[ELASTIC_IP]/32
+{{</ file >}}
+
+    If you configured more than one Elastic IP on your Linode, you can add additional interface entries to your network interfaces configuration file as follows:
+
+    {{< file >}}
+Address=[ELASTIC_IP]/32
+Address=[ELASTIC_IP]_2/32
+{{</ file >}}
+
+    **CentOS 8**
+
+    Edit your Linode's `/etc/sysconfig/network-scripts/ifcfg-eth0` file with the following entry. Replace `[ELASTIC_IP]` with the Elastic IPv4 address:
+
+    {{< file >}}
+IPADDR1=[ELASTIC_IP]
+PREFIX1="32"
+{{</ file >}}
+
+    If you configured more than one Elastic IP on your Linode, you can add additional interface entries to your network interfaces configuration file as follows:
+
+    {{< file >}}
+IPADDR1=[ELASTIC_IP]
+PREFIX1="32"
+
+IPADDR2=[ELASTIC_IP]_2
+PREFIX2="32"
+{{</ file >}}
+
+1.  Apply the `eth0` network interface configuration:
+
+    **Debian 10, Ubuntu 18.04 & CentOS 8**
+
+        sudo ifdown eth0 && sudo ifup eth0
+    **Ubuntu 20.04**
+
+        systemctl restart systemd-networkd
+
```

Comment:
@wildmanonline should we include instructions for applying the network config with CentOS too?

Reply:
I think those are handled with `sudo ifdown eth0 && sudo ifup eth0` in the section above that includes CentOS. I renamed that section's bold psuedo-header to "Debian, Ubuntu (18.04 and 16.04), and CentOS/RHEL"